src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditContainer.tsx

```diff
@@ -73,7 +73,24 @@ const ProjectEditContainer: React.FC<ProjectEditContainerProps> = ({ project, ex
   const query = useQuery();
   const allUsers = useAllUsers();
   const toast = useToast();
-  const { slideDeckLink, bomLink, gDriveLink, taskListLink, name, budget, summary } = project;
+  const { name, budget, summary } = project;
+
+  const projectLinks = project.links.map((link) => {
+    return {
+      linkId: link.linkId,
+      url: link.url,
+      linkTypeName: link.linkType.name
+    };
+  });
+
+  const linkTypes = projectLinks.map((link) => link.linkTypeName);
+
+  Object.values(RequiredLinkType).forEach((linkType) => {
```

Comment:
is this adding the required link type if it's not there? kinda weird, I'd find a better way to enfore that the required types are included (maybe in yup) or at least put a comment saying what you're doing here

Reply:
yes that is what this is doing. its just supplying default values, not necessarily enforcing them